### PR TITLE
Add ACL details to documentation

### PR DESCRIPTION
**Pull request by sravan-s**
Fixes: https://github.com/ZEPL/zeppelinhub/issues/2856

- [x] Add details about ACL into documentation

### CODE REVIEW

File: docs/sharing_notebooks.md [hunk 1]
```diff
@@ -24,16 +24,45 @@ Now you're an owner of this Space. You can invite collaborators and manage the s
 <img src="../img/after_add_space.png" class="image-box big-img"/>
 
 <br/ >
-## Sharing Notebooks to the Space
-You can share/[publish the notebook](exploring_notebook.md) in **Shared** main page. 
+## Sharing Notebooks
+Notebooks can be shared to a space or can be shared directly to another
+user, or you can [publish the notebook](exploring_notebook.md) to the web.
+<br />
+You can share/ in **Shared** main page.
```
Comment (minahlee) [inline docs/sharing_notebooks.md]: `Shared` menu renamed to `Spaces`. Can you update it?

**Review by minahlee — APPROVED** [review]
I left one comment. Except that LGTM

File: docs/sharing_notebooks.md (hunk 1 above)
Comment (sravan-s) [reply docs/sharing_notebooks.md]: Yes, sure

File: docs/sharing_notebooks.md [hunk 2]
```diff
@@ -24,16 +24,45 @@ Now you're an owner of this Space. You can invite collaborators and manage the s
...
+**Permission**
+
+* View - Allows others to only view the notebook
+* Edit - Allows shared party to edit the notebook(If you share notebooks
+with this option enabled, your interpreter will be used when the
+notebook is executed)
+
+**Format**
+
+Same as three visual modes in Zeppelin
```
Comment (AhyoungRyu) [inline docs/sharing_notebooks.md]: It should be `Apache Zeppelin` instead of `Zeppelin`

File: docs/sharing_notebooks.md [hunk 3]
```diff
@@ -24,16 +24,45 @@ Now you're an owner of this Space. You can invite collaborators and manage the s
 <img src="../img/after_add_space.png" class="image-box big-img"/>
 
 <br/ >
-## Sharing Notebooks to the Space
-You can share/[publish the notebook](exploring_notebook.md) in **Shared** main page. 
+## Sharing Notebooks
+Notebooks can be shared to a space or can be shared directly to another
+user, or you can [publish the notebook](exploring_notebook.md) to the web.
```
Comment (AhyoungRyu) [inline docs/sharing_notebooks.md]: It was my bad. Could you change `exploring_notebook.md` to `exploring_notebooks.md`?

File: docs/sharing_notebooks.md [hunk 4]
```diff
@@ -24,16 +24,45 @@ Now you're an owner of this Space. You can invite collaborators and manage the s
...
+
+* Default
+* Simple
+* Report
+
+You can allow the shared party to access your notebooks only in the specified visual mode
+
+**Version**
+
+ZEPL can leverage the version control system system as in Zeppelin,
```
Comment (AhyoungRyu) [inline docs/sharing_notebooks.md]: There are double `system` :) And also, need to change `Zeppelin` to `Apache Zeppelin`

**Comment by sravan-s** [comment]
@AhyoungRyu Changed as per you requested, 
I have a question, are there any lint rules like 80 chars. per line for documentation?

**Comment by AhyoungRyu** [comment]
No we don't but for the sake of better readability, it'll be better to add newline if you feel the sentence is too long :)

**Review by AhyoungRyu — APPROVED** [review]

**Comment by sravan-s** [comment]
Merging if there are no more comments

**Pull request merged**